lms/templates/emails/activation_email.html

```diff
@@ -0,0 +1,25 @@
+<%inherit file="main.html" />
+<%! from django.utils.translation import ugettext as _ %>
```

Comment:
Unused import can be deleted

Reply:
You are right, I will remove this import